wiki/Front-end_with_curl.md

```diff
@@ -0,0 +1,294 @@
+# Curl Examples to front-end
+
+The front-end is accessible not only via the web GUI.
+
+## Introduction
+
+The front-end has a bridge functionality that is built in as a part of the front-ends web server and handles request coming from the web GUI towards any of the back-ends configured. The front-end may also be used by other tools such as CURL or any kind of program that can make HTTP(S) requests.
+
+#### The front-end handles:
+* POST, PUT and GET requests
+* Default back-end
+* Input back-end
+
+While the web GUI may use the back-end instances list and select different back-ends in an easy way, a user that uses the front-end without the included web GUI may need to specify a back-end URL. This may be done by adding a `backendurl` parameter to the request.
+
+Note that for users where the front-end and back-end is running with the help of Tomcat there will be context paths used.
+
+### Default back-end
+
+The front-end may have been setup with a default back-end, this means that if no back-end is specified when making a HTTP request to the front-end the default back-end will be used.
+If you want to see a list of back-ends and see if there is a default back-end set you may use the command:
+
+###### Normal curl command for get:
+
+    curl -X GET http://*front-end-url*/*contect-path-if-any*/backend
+
+###### Curl command for get with host, port using Tomcat:
+
+    curl -X GET http://localhost:8080/eifrontend/backend
+
+###### Curl command using EI default settings:
+
+    curl -X GET http://localhost:8080/backend
+
+
+The default back-end should have the key `defaultBackend` set to `true`. If the JSON list ends up empty there are no back-ends specified in the front-end. If there is no JSON object with the key set to true there is no default back-end.
+
+You may see that in the list, all objects have a key `active` set to `true or false`, this key points to the back-end that acts as default, if no default back-end exist, the active should usually be the first object in the list.
+
+If the back-end list lacks a default back-end one may be added by using a HTTP POST request. The injected object should be specified in JSON and look like the example below.
+
+    [
+        {"name":"Any default name", "host":"*Your back-end url*", "port":8090, "contextPath":"", "https":false, "defaultBackend":true}
+    ]
+
+You may add a back-end using for example CURL:
+
+    curl -d '{"name":"My Default Back-End","host":"localhost","port":8090,"contextPath":"","https":false,"defaultBackend":true}' \
+        -H "Content-Type: application/json" -X POST http://localhost:8080/*front-end-context-path*/backend
+
+__Note:__ A back-end names must be unique.
```

Comment:
I think that the plural form is the only one that makes sense here. Also extreme nitpick but I believe that first letter in this case should not be capitalized (https://www.grammarly.com/blog/capitalization-after-colons/).

Reply:
I think that if "Note" is part of the sentence then yes the first letter should not be capitalized. But if it is used as for example "Warning:" or "Example:" then it should be capitalize.

Reply:
Done.